Services/Administration/OJS.Services.Administration.Business/ProblemGroups/ProblemGroupsBusinessService.cs

```diff
@@ -171,6 +171,18 @@ public ICollection<ProblemGroupDropdownModel> GetOrderByContestId(int contestId)
                 .MapCollection<ProblemGroupDropdownModel>()
                 .ToHashSet();
 
+        public double GetNewLatestOrderByContest(int contestId)
+        {
+            var lastProblemGroupOrderBy = this.problemGroupsData.GetLastNonDeletedByContest(contestId);
+
+            if (lastProblemGroupOrderBy == null)
```

Comment:
Are you sure this will be `null`? Could it be `0` if no problem group is found for that contest?

Reply:
Fixed